sp800-63-3/sec5_DIRM.md

```diff
@@ -157,11 +157,11 @@ The statement SHALL include, at a minimum:
 
 ### 5.6 Migrating Identities
  
-As these guidelines are revised, CSPs SHALL consider how changes in requirements affect their user population. In some instances the user population will be unaffected, yet in others, the CSP will require users undergo a transitional activity. For example, request users, upon initial logon since last revision, supply additional proofing evidence to adhere to new IAL requirements. This SHALL be a risk-based decision, made in context of the CSP, any RPs that use the CSP, mission, and the population served. The following considerations serve only as a guide to agencies when considering the impacts of requirements changes:
+As these guidelines are revised, CSPs SHALL consider how changes in requirements affect their user population. In some instances, the user population will be unaffected, yet in others, the CSP will require users undergo a transitional activity. For example, CSPs may request users, upon initial logon since last revision, to supply additional proofing evidence to adhere to new IAL requirements. This SHALL be a risk-based decision, made in context of the CSP, any RPs that use the CSP, mission, and the population served. The following considerations serve only as a guide to agencies when considering the impacts of requirements changes:
```

Comment:
update to "CSPs may request users - upon initial logon since last revision - supply additional proofing evidence "